mmseg/models/backbones/fast_scnn.py

```diff
@@ -23,6 +23,8 @@ class LearningToDownsample(nn.Module):
             dict(type='BN')
         act_cfg (dict): Config of activation layers. Default:
             dict(type='ReLU')
+        dw_act_cfg (dict):Activation config of depthwise ConvModule. If it is
+            'default', it will be the same as `act_cfg`. Default: None.
```

Comment:
In `DepthwiseSeparableConvModule`, if it is 'default', it will be the same as `act_cfg`.